packages/lib-utils/src/components/horizontal-nav/HorizontalNav.tsx

```diff
@@ -0,0 +1,52 @@
+import { Tabs, Tab, TabTitleText } from '@patternfly/react-core';
+import React from 'react';
+
+export type Tab = {
```

Comment:
It would be good to have a URL path so that users can link to specific tabs (and we'd need this if we wanted to swap out the tab implementation for HAC core's in OpenShift console).